stdlib/3.7/dataclasses.py

```diff
@@ -0,0 +1,47 @@
+from typing import *
```

Comment:
Thanks for adding these stubs!

A couple of style comments:
- don't use `import *`
- names that don't actually exist at runtime (like `T` and `DictType`) should be private, so their names should start with an underscore.
- indent with four spaces
- add spaces after `:` in annotations and around `=` for annotated functions with arguments.